pydis_site/apps/content/resources/guides/python-guides/windows/common-issues.md

```diff
@@ -0,0 +1,164 @@
+    > virtual environment, it should automatically activate it when you open a new terminal, so running the
+    > `pip install` command in the VSCode terminal should work. If you are not using a virtual environment, you can use
+    > the `py` launcher to specify the installation you want to install to.
+
+### I `pip` installed a command line program but it isn't recognized
+
+You pip installed a command line program like [`pyinstaller`](https://pypi.org/project/pyinstaller/) or
+[`black`](https://pypi.org/project/black/) that is intended to be run from the command line, but it does not work. Two
+common causes for this are:
+
+-   You are using Python from the Microsoft Store ([which we don't recommend](../miscrosoft-store)).
+
+    To check if this is the case, type `pip -V`. If the path output includes something like
+    `PythonSoftwareFoundation.Python.3.10_3.10.2288.0_x64__qbz5n2kfra8p0`, you are using Python from the Microsoft
+    Store. To fix this you will either have to use the full path to the scripts you want to use, or [uninstall the
+    Microsoft Store Python](/static/images/content/python-on-windows/ms_store_uninstall.png) and [install Python
+    properly](../installing-and-using-python) with a [python.org installer](https://www.python.org/downloads).
+
+-   You manually modified your PATH Environment Variable incorrectly
+
+    If you added `Python` to PATH manually through environment variables, it is possible that you only added the
+    executable but not the `\Scripts` folder. This would cause issues with `pip` and make command line tools not
+    accessible. Look at our [guide on adding Python to the
+    PATH](../putting-python-on-path/#advanced-method-manually-edit-the-path) for how to verify and fix this.
+
+Projects that add a command line program usually work by adding an executable to the `\Scripts` folder of your Python
+install (the full path of which is `C:\Users\<user>\AppData\Local\Programs\Python\Python311\Scripts` in a default 3.11
+install). If you checked "Add python.exe to PATH" when installing Python, this folder will have been added to PATH, so
+anything there should be runnable directly from the command line. Virtual environments also have their own `\Scripts`
+directories that things can be installed to and are effectively on the PATH when the venv is properly activated.
+
+### I get a `SyntaxError` when trying to run `pip`,`python`, or another command.
+
+When you type `python` in the terminal you enter into the python REPL ([read-evaluate-print
+loop](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop)), also known as the Python console. This lets
+you run `Python` code line by line without having to create a file. You also may have opened a Python REPL by clicking
+on the Python App in your Start Menu.
+
+General commands like `pip` and `python` should be run in the normal terminal, not in the Python REPL. It usually has
```

Comment:
I think this could cause confusion for beginners, because I might ask a question like "normal" means here.
So maybe it'll be better if we be more specific by saying it's the windows terminal, e.g. command prompt

Reply:
Good point. I'll clarify that "normal" probably means Command Prompt or PowerShell.